target/linux/generic/config-6.12

```diff
@@ -2998,6 +3041,10 @@ CONFIG_KUSER_HELPERS=y
 # CONFIG_KVM_AMD_SEV is not set
 # CONFIG_KVM_GUEST is not set
 # CONFIG_KVM_INTEL is not set
+# CONFIG_KVM_INTEL_PROVE_VE is not set
+CONFIG_KVM_MAX_NR_VCPUS=1024
+# CONFIG_KVM_PROVE_MMU is not set
+# CONFIG_KVM_SW_PROTECTED_VM is not set
```

Comment:
These KVM symbols are x86 only. I'd prefer to remove them from the generic config.

Reply:
Done